packages/driver/test/cypress/integration/commands/misc_spec.coffee

```diff
@@ -69,6 +69,17 @@ describe "src/cy/commands/misc", ->
       cy.wrap({}).then (subject) ->
         expect(subject).to.deep.eq {}
 
+    it "cy.wrap(undefined) should retry", () ->
+      spy = cy.spy()
+
+      cy.wrap().should ->
+        spy()
```

Comment:
add comments here, and change to `cy.stub()`

Reply:
:+1: